src/assets/javascript/showPassword.js

```diff
@@ -1,5 +1,5 @@
-window.GOVUK = window.GOVUK || {};
-window.GOVUK.Modules = window.GOVUK.Modules || {};
+window.GOVSignIn = window.GOVSignIn || {};
```

Comment:
Is this the only place this gets initalised in the app?
Or are there other GOVUK Module elements we'll want to clear up later?

Reply:
I think the `GOVUK.Modules` thing is legacy from the days the team used to be on GOUVK as that is the standard namespacing there. This is the only place where it's referenced